### DISCUSSION TITLE
Bom and pick and place files

### DISCUSSION THREAD

**Discussion by ogg1e** [frntc/RAD#3] [Ideas]
Any chance you could generate and upload the bom and  pick and place files for PCB assembly?

**Comment by frntc** [comment]
I hope you understand that I’m reluctant to provide BOM and CPL files, as this makes it even easier for people who ignore licenses and sell hardware on internet marketplaces (which I find somewhat frustrating). I prefer if the hardware Is available for fair prices (for those who can’t/don't want to build it themselves) from people who I know are fair traders. If you’re looking for that: join forum64.de or visit restore-store.de (not my shop, contact him at shop@restore-store.de) if you don’t find what you’re looking for)

**Comment by ogg1e** [comment]
I totally get your concern. I've seen many times that sort of thing happen. I was just going to make a batch of 5 and give them to my friends.
